amethyst_ui/src/prefab.rs

```diff
@@ -170,13 +158,17 @@ impl<'a> PrefabData<'a> for UiTransformBuilder {
         }
         system_data.0.insert(entity, transform)?;
         if self.mouse_reactive {
```

Comment:
should rename `mouse_reactive` here

Reply:
That would be a breaking change. I plan on removing the field from UiTransform in the next PR.

Reply:
Understandable